Util/function/build.gradle

```diff
@@ -5,6 +5,8 @@ plugins {
 
 dependencies {
     api project(':qst-type')
+    api project(':engine-query-constants')
+    api project(':engine-time')
```

Comment:
I don't like this dependency on engine-time, given it is a bit more sprawling. Does `ToChunkTypeTransform` living in `engine-time` work for you?

Reply:
I think that should work, yes